lib/apiservers/portlayer/swagger.yml

```diff
@@ -819,6 +819,51 @@ paths:
           description: "Error"
           schema:
             $ref: "#/definitions/Error"
+  /containers/{id}/logs:
+    get:
+      description: "Gets the container logs by id"
+      summary: "Gets the container logs"
+      operationId: GetContainerLogs
+      tags: ["containers"]
+      consumes:
+        - application/octet-stream
+      produces:
+        - application/octet-stream
+      parameters:
+        - name: id
+          in: path
+          type: string
+          required: true
+        - name: follow
+          in: query
+          type: boolean
+          default: false
+          required: false
+        - name: since
+          in: query
+          type: integer
+          format: int64
+          required: false
+        - name: timestamp
```

Comment:
This is a docker'ism, right?  For our PL, we don't have to use their (docker's naming).  I'd rather call this "raw".  We could then JSONify the log output and include timestamps in the body unless raw=1.

Just a thought.